frontend/typedoc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://typedoc.org/schema.json",
+  "out": "src/generated/docs",
+  "plugin": ["typedoc-plugin-markdown"],
+  "entryPointStrategy": "expand",
+  "exclude": ["src/components/**/*", "src/features/**/components/**/*", "src/stories/**/*"],
```

Comment:
Why is this excluded?

Reply:
`src/stories/**/*` is indeed an error. I copied it from my previous attempts.
`src/components/**/*` and `src/features/**/components/**/*` because I think there should only be one documentation of a component. They should be documented in the stories and not by TypeDoc.
Also the component documentation of TypeDoc is quite ugly.

TypeDoc
![image](https://user-images.githubusercontent.com/9990578/232231375-4f343c79-e4e7-41fd-b0cf-6e87bbccbbd5.png)
vs
Storybook
![image](https://user-images.githubusercontent.com/9990578/232231694-d722592e-e962-4184-a87b-8e70c44eac6a.png)